Makefile

```diff
@@ -3,7 +3,7 @@ ifneq ($(DEBUG_MODE),true)
 endif
 
 # NOTE: BACKEND is dup in include/common.sh to allow BACKEND override when loading from json config files
-BACKEND?=kind
+export BACKEND?=kind
```

Comment:
I cannot remember correctly now - have to check that out. But IIRC we had to remove the export here for the JSON config file support.

Reply:
So it looks like: https://github.com/SUSE/catapult/commit/3696b195b1881d2c1670a360e445fba5674a345b . The export is happening in common/ so the JSON parameters can work. Why adding it back here?

Reply:
I exported BACKEND globally so it can be read from modules/common/deps.sh: https://github.com/SUSE/catapult/pull/98/files#diff-8ef793e96cb603b21d1eb01d85cab069R3-R5

We need BACKEND to do line 3:

```
line3   . "$ROOT_DIR/backend/$BACKEND/defaults.sh"
line4   . ./defaults.sh                             
line5   . ../../include/common.sh
```